.github/workflows/staging-sgx-restart.yml

```diff
@@ -0,0 +1,37 @@
+name: Staging SGX parachain
+
+on:
+  workflow_dispatch:
+    inputs:
+      action:
+        type: choice
+        description: Select the action
+        required: true
+        default: restart
+        options: 
```

Comment:
`choice` is better if we only allow one action each time.

maybe we only need two options:
- stop only
- stop and start